lib/heap_sort.rb

```diff
@@ -1,8 +1,18 @@
-
+require_relative 'min_heap'
 
 # This method uses a heap to sort an array.
-# Time Complexity:  ?
-# Space Complexity: ?
+# Time Complexity:  O(n log n) where n is the number of elements in the given array
+# Space Complexity: O(n)
 def heap_sort(list)
```

Comment:
This turns the array into a heap, it doesn't "sort" the array  using a heap.  So it's not quite the exercise.